internal/generator/vector/output/loki/cert-key.go

```diff
@@ -0,0 +1,18 @@
+package loki
+
+import (
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/output/security"
+)
+
+type TLSKeyCert security.TLSCertKey
+
+func (kc TLSKeyCert) Name() string {
+	return "elasticsearchCertKeyTemplate"
```

Comment:
please change it to `lokiCertKeyTemplate`

Reply:
changed